dubbo-cluster/src/main/java/com/alibaba/dubbo/rpc/cluster/loadbalance/ConsistentHashLoadBalance.java

```diff
@@ -32,114 +33,156 @@
 
 /**
  * ConsistentHashLoadBalance
- *
+ * Fixed by:  yunfei.gyf
+ * After fixed, the ConsistentHashLoadBalance will surpport to used in GenericService,
+ * and it will reduce the times of rebuilding hash cycle when the provider restart 
+ * but nothing to change
+ * 
+ * @author yunfei.gyf

```

Comment:
Please remove the author's information, git will records your contribution faithfully. More Info, see #1044

Reply:
It has been removed